airflow/providers/sftp/operators/sftp.py

```diff
@@ -134,29 +176,68 @@ def execute(self, context: Any) -> str:
 
             with self.ssh_hook.get_conn() as ssh_client:
                 sftp_client = ssh_client.open_sftp()
-                if self.operation.lower() == SFTPOperation.GET:
-                    local_folder = os.path.dirname(self.local_filepath)
-                    if self.create_intermediate_dirs:
-                        Path(local_folder).mkdir(parents=True, exist_ok=True)
-                    file_msg = f"from {self.remote_filepath} to {self.local_filepath}"
-                    self.log.info("Starting to transfer %s", file_msg)
-                    sftp_client.get(self.remote_filepath, self.local_filepath)
+                if self.local_filepath and self.remote_filepath:
+                    if isinstance(self.local_filepath, list) and isinstance(self.remote_filepath, str):
+                        for file_path in self.local_filepath:
+                            local_folder = os.path.dirname(file_path)
+                            remote_folder = os.path.dirname(self.remote_filepath)
+                            local_file = ntpath.basename(file_path)
```

Comment:
Why `ntpath` specifically, and only this function?

Reply:
I needed to take the name of file from full path, this function was just right for this task :)

Reply:
But why not `os.path.basename`? `ntpath` splits on backslashes (as it should) and that would cause issues on Linux (where ``\`` is a valid character in file name).

Reply:
![image](https://user-images.githubusercontent.com/86562687/126042276-6861f2cd-ae0f-4b70-a5cc-74d4f5a25ceb.png)
its difference between functions

Reply:
You have mistake in your second comparision (missing "txt" in second example) - that's why you have difference

Reply:
you right, tnx